service/storage_service.cc

```diff
@@ -5250,7 +5250,7 @@ future<> storage_service::load_tablet_metadata() {
     }, acquire_merge_lock::no);
 }
 
-future<> storage_service::process_tablet_split_candidate(table_id table) {
+future<> storage_service::process_tablet_split_candidate(table_id table) noexcept {
```

Comment:
Why noexcept?

Reply:
monitor expects this to not throw, so it will not be aborted. this is only about documenting it, since in practice, it shouldn't throw.

Reply:
shouldn't throw = will not propagate exception